ehrapy/anndata/anndata_ext.py

```diff
@@ -381,8 +381,9 @@ def set_numeric_vars(
 
     vars_idx = _get_column_indices(adata, vars)
 
-    for i in range(n_values):
-        adata.X[:, vars_idx[i]] = values[:, i]
+    adata.X[:, vars_idx] = values
```

Comment:
this is a significant speedup, 2-4x with dask on my machine

Reply:
Lovely!